doc/start/os-recommendations.rst

```diff
@@ -22,6 +22,9 @@ Linux Kernel
   *at least* 4.19-based "longterm maintenance" kernel series.  If you can
   use a newer "stable" or "longterm maintenance" kernel series, do it.
 
+  For KRBD, a kernel of *at least* version 5.4 is the minimum necessary for
+  reasonable RBD feature flag support.
```

Comment:
I'd move this to the "Older kernel client versions may not support ..." paragraph below.